src/components/core/update/updateSlides.js

```diff
@@ -2,6 +2,27 @@ import { getWindow } from 'ssr-window';
 import { extend } from '../../../utils/utils';
 
 export default function updateSlides() {
+  const getDirectionLabel = (property) => {
+    const swiper = this;
```

Comment:
Let's move `const swiper = this` above the `const getDirectionLabel`, because right below we have same `const swiper = this` in upper scope